udata/core/dataset/forms.py

```diff
@@ -106,6 +106,8 @@ class DatasetForm(ModelForm):
     model_class = Dataset
 
     title = fields.StringField(_('Title'), [validators.required()])
+    acronym = fields.StringField(_('Acronym'),
+        description=_('An optionnal short title or acronym'))
```

Comment:
Idem. Do we want to promote "short titles"? What could those be if not an acronym?

Reply:
Good question. I think I'll keep only acronym for the wording right now

Reply:
👌